dms/template/portal.xml

```diff
@@ -23,27 +23,30 @@
                     <t t-set="placeholder_count" t-value="'dms_directory_count'" />
                 </t>
             </div>
-        </xpath>
+        </div>
     </template>
     <template
         id="portal_my_dms_breadcrumbs"
         name="My DMS: Breadcrumbs"
         inherit_id="portal.portal_breadcrumbs"
     >
         <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
+            <li
+                id="portal_breadcrumbs_documents"
+                t-if="page_name == 'dms_directory' or dms_directory"
+                t-attf-class="breadcrumb-item #{'active 'if not dms_directory else ''}"
+            >
+                <a
+                    t-if="dms_directory"
+                    t-attf-href="/my/dms?{{ keep_query() }}"
+                >Documents</a>
+                <t t-else="">Documents</t>
+            </li>
```

Comment:
These changes are not necessary and it should be as before. The breadcrum will only be shown if access_token is not used (if it is used, probably there is no logged in user and adding the link to “/my/dms” does not make sense because it will not work).